Gemfile

```
@@ -1,2 +1,8 @@
 source "https://rubygems.org"
-gemspec
\ No newline at end of file
+
+gem "jekyll", "~> 4.2.0"
+gem "jekyll-feed", "~> 0.15"
+gem "jekyll-seo-tag", "~> 2.7"
+gem "rouge", "~> 3.23"
+gem "webrick", "~> 1.7"
+gem "jekyll-remote-theme"
```

RuboCop (1.81.7)

[convention] Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `jekyll-remote-theme` should appear before `webrick`.

(Bundler/OrderedGems)